alternator/executor.cc

```diff
@@ -96,6 +96,44 @@ std::string make_jsonable::to_json() const {
     return rjson::print(_value);
 }
 
+
+static bool is_leaf(const rjson::value& value) {
+    return !value.IsObject() && !value.IsArray();
+}
```

Comment:
I suggest moving this generic rapidjson code which isn't specific to Alternator to `utils/rjson.{hh,cc}`.
This will slightly de-clutter the already huge executor.cc, but more importantly, will allow using this important function in other places as well. You already defined destroy_gently() as extern, let's just put it the right file and declare it in utils/rjson.hh.

Reply:
ok